app/views/docs/getting-started-for-server.phtml

```diff
@@ -401,7 +404,11 @@ promise.then(function (response) {
         <h3>PHP</h3>
 
         <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP SDK">
-            <pre class="line-numbers"><code class="prism language-php" data-prism>$users = new Users($client);
+            <pre class="line-numbers"><code class="prism language-php" data-prism>use Appwrite\Client;
```

Comment:
I think you're still importing an unused `Client` class here and in other examples. Please make sure the imports are concise and consistent.

Reply:
Sorry, I missed that completely.
It should now be removed everywhere.